packages/mvsp/mvsp-alpha.en.asciidoc

```diff
@@ -31,10 +32,10 @@ h| Description
 * Reasonable restrictions on testing can be defined
 
 | 1.3 Self-assessment
-| Perform annual (at a minimum) security self-assessments using the latest MVSP release
+| Perform and publish security self-assessments using the latest MVSP release at least annually
```

Comment:
```suggestion
| Perform and make available security self-assessments using the latest MVSP release at least annually
```

We've seen pushback on specific language around publication in the past. Make available seems to be a reasonable mid-point for companies with a more closed culture of broad public sharing. This is in-line with the language in sub-processor control 4.3.